src/odemis/util/test/units_test.py

```diff
@@ -99,6 +99,8 @@ def test_readable_str(self):
                   ((999.4, "V", 3), "999 V"),
                   ((200e-6, "m"), u"200 µm"),
                   ((0.0, "m"), "0 m"),
+                  ((0, "rad"), "0 rad"),
+                  ((0, "rad", 3), "0 rad"),
```

Comment:
You could add a test with a decimal instead of `0`. Or is this something we never use? What about `mrad` - do we display this?

Reply:
We have plenty of tests with decimals. I wanted more tests with 0, as it's a bit special (ie, very tiny value, but no unit prefix).
The GUI does disaplay the prefix in some cases (and that's why I originally wrote this test case), but this is directly handled into the number-text widget.